templates/pages/checkout.html

```diff
@@ -1 +1,35 @@
-brand.html
+<!doctype html>
+<html>
+<head>
+    <base href="/">
+    <meta charset="UTF-8"/>
+    <meta name="keywords" content="checkout"/>
+    <meta name="description" content="Bigcommerce checkout"/>
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="viewport" content="width=device-width">
+    <link rel="SHORTCUT ICON" href="{{ FaviconUrl }}" />
```

Comment:
Favicon should already be included by `empty.html`. Also, there's already `head.favicon` Stencil object, so we probably don't need another one.